src/Symfony/Bridge/Twig/Tests/Command/DebugCommandTest.php

```diff
@@ -264,12 +265,12 @@ public function getDebugTemplateNameTestData()
 Configured Paths
 ----------------
 
- ----------- -------------------------------------- 
-  Namespace   Paths                                 
- ----------- -------------------------------------- 
+ ----------- --------------------------------------
+  Namespace   Paths
+ ----------- --------------------------------------
   @Twig       templates/bundles/TwigBundle%e%A
               vendors/twig-bundle/Resources/views%e%A
- ----------- -------------------------------------- 
+ ----------- --------------------------------------
```

Comment:
whistespace in this file should be kept

Reply:
Yep need to double check everything again before merge

Reply:
Fixed